lib/ui/events.js

```diff
@@ -0,0 +1,317 @@
+        track : {
+          style : { bg: 'grey' },
+        }
+      },
+      style : {
+        label  : { bold: true },
+        border : { fg: 'white' },
+        header : { fg: 'grey' },
+        cell   : { fg: 'white', selected: { bg: 'blue' } },
+      }
+    });
+
+
+    // do not render when detached. 
+    events_table._orender = events_table.render;
+    events_table.render = function () {
+      if (this.parent == null) {
+        return;
+      }  
+      this._orender();
+    };
+
+    // work around to preserve selection and scrolling
+    events_table.on('detach', () => {
+      const { selected, childBase, childOffset } = events_table;
+      events_table.once('attach', () => {
+        events_table.select(selected);
+        Object.assign(events_table, { childBase, childOffset });
+        events_table.scrollTo(selected);
+      });
+    });
+
+    events_table.on('key C-z', () => {
+      cancellations.run(key);
+      navbar.remove(key);
+    });
+
+    const { promise, cancellation } = client.events(namespace).get({ cancellable : true });
+    cancellations.add(key, cancellation);
+    until(promise)
+    .spin(s => events_table.setLabel(`${s} ${key}`))
+    .succeed(_ => events_table.setLabel(`${key}`))
+    .then(response => {
+      const events_list = JSON.parse(response.body.toString('utf8'));
+      events_table.resource_version = events_list.metadata.resourceVersion;
+      this.fill_events_table(events_list, events_table, cancellations, key);
+    })
+    .then(_ =>{
+      const { promise, cancellation } = client.event(namespace)
+      .watch(events_table.resource_version)
+      .get({ generator: update_events_table.bind({ events_table, debug, highlightEvent })})
+      cancellations.add(key, cancellation);
+      return promise;
+    }).catch(error => {
+      debug.log(`{red-fg}Error creating Events tab: ${error}{/red-fg}`);
+      if (error.response.statusCode == '403') {
+        const unauthorized_msg = `{red-fg}It seems you don't have the rights to fetch ${key}. Please contact your Cluster administrator.{/red-fg}`;
+        debug.log(unauthorized_msg);
+        events_table.setData([[unauthorized_msg]]);
+      }
+    });
+
+    events_table.on('select', (item, i) => {
+      if (i === 0 || events_table.uid === events_table.rows[i].uid) return;
+      this.describe_event(i);
+    });
+
+    this.describe_event = function(i) {
+      event_description.setContent("");
+      const namespace = events_table.rows[i].namespace;
+      const name = events_table.rows[i].name;
+      events_table.uid = events_table.rows[i].uid;
+      const { promise, cancellation } = client.describe_event(namespace, name).get({ cancellable : true });
+      cancellations.add(key, cancellation);
+      until(promise)
+      .spin(s => event_description.setLabel(`${s} Event`))
+      .succeed(_ => event_description.setLabel(`Event ${events_table.rows[i][2]} {inverse}${events_table.rows[i][4]}{/inverse}`))
```

Comment:
I'd suggest to follow current pattern used to label widgets, e.g., `Event [name]`.